api/src/test/java/org/openmrs/module/fhir2/api/search/MedicationRequestSearchQueryTest.java

```diff
@@ -497,12 +518,10 @@ public void searchForMedicationRequest_shouldReturnMedicationRequestByParticipan
 		IBundleProvider results = search(theParams);
 		
```

Comment:
This whole test class was kind of strange... each of the tests I modified below first tested that there were the proper number of elements in the result set, but then called a utility get method that fetched the first ten elements in that result set... and then confirmed that return list did have, indeed, 10 elements.

I changed the utility method to return all the elements (see below) and just changed it to  only test once that the proper number of elements were present (generally  2 less than previously because of the 2 DISCONTINUE orders in the test dataset)